shared/src/packages/trucks/libs/validation-schemas/validation-schema.ts

```diff
@@ -0,0 +1,4 @@
+export { truckCreateRequestBody } from './truck-create-request-body.validation-schema.js';
+export { truckIdParameter } from './truck-id-parameter.validation-schema.js';
+export { LICENSE_PLATE_NUMBER_REGEX } from './truck-regular-expressions/truck-regular-expressions.js';
```

Comment:
i'm not sure that you will use LICENSE_PLATE_NUMBER_REGEX somewhere except validation schema, so you can just remove unnecessary export here, and also remove from exports in trucks/trucks file

Reply:
I have validation on the fe for fetching truck objects from a form, and this common regex is used there for validating the license plate number.

Reply:
Do you mean that I should move all validation to the shared folder?